.github/workflows/docker-nightly.yml

```diff
@@ -0,0 +1,22 @@
+name: php-nightly
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y%m%d')"
+    - uses: docker/build-push-action@v1
+      with:
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+        registry: docker.pkg.github.com
+        repository: "${{ github.repository }}/php-nightly"
```

Comment:
Let's make this `php-master-nightly`, so we have room for `php-7.4-nightly` etc.

Reply:
@nikic agree, it'd be possible to build using the same workflow on multiple branches. Any thoughts about supporting alpine as well?